apps/voting/contracts/Voting.sol

```diff
@@ -363,36 +476,42 @@ contract Voting is IForwarder, AragonApp {
         emit StartVote(voteId, msg.sender, _metadata);
 
         if (_castVote && _canVote(voteId, msg.sender)) {
-            _vote(voteId, true, msg.sender);
+            _vote(voteId, true, msg.sender, false);
         }
     }
 
     /**
     * @dev Internal function to cast a vote or object to.
       @dev It assumes that voter can support or object to the vote
     */
-    function _vote(uint256 _voteId, bool _supports, address _voter) internal {
+    function _vote(uint256 _voteId, bool _supports, address _voter, bool _isManager) internal {
         Vote storage vote_ = votes[_voteId];
 
         // This could re-enter, though we can assume the governance token is not malicious
         uint256 voterStake = token.balanceOfAt(_voter, vote_.snapshotBlock);
```

Comment:
LDO balance calculation can be transferred after checking that the tokenholder has already voted (to str 499)

Reply:
Nice catch

Reply:
`token.balanceOfAt` is called twice and this is quite complex function inside MiniMe token, for example take a look into relatively good case second call is cost us additional 2% of gas consumption. Take a look here: https://dashboard.tenderly.co/tx/mainnet/0xc20bba17d0809089e00ffb7e69b870f5659ae0ddc296ecf7906e8419e3b56a75/gas-usage

I am suggest to consider possibility of moving `_hasVotingPower` check into the `_vote`

Reply:
Addressed in 5d01f379ce314fcc413316738903b657d75f85f6.

Reply:
We've decided to move `token.balanceOfAt` call up and convert `votingPower` to `_vote`'s argument,